src/lib/connectors.ts

```diff
@@ -15,6 +15,7 @@ const POLLING_INTERVAL = 12000;
 const RPC_URLS: { [chainId: number]: string } = {
   1: process.env.REACT_APP_RPC_URL_1 as string,
   4: process.env.REACT_APP_RPC_URL_4 as string,
+  42: process.env.REACT_APP_RPC_URL_42 as string,
```

Comment:
Can we update [the readme's example .env](https://github.com/zer0-os/ZNS-dapp#developing-locally) with this new environment variable and its example value.